x-pack/plugins/cloud_security_posture/public/components/csp_timestamp_table_cell.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import moment, { type MomentInput } from 'moment';
+import { EuiToolTip } from '@elastic/eui';
+
+export const CspTimestampTableCell = ({ timestamp }: { timestamp: MomentInput }) => (
+  <EuiToolTip position="top" content={moment(timestamp).format('MMMM D, YYYY @ HH:mm:ss.SSS')}>
+    <span>{moment(timestamp).fromNow()}</span>
+  </EuiToolTip>
+);
```

Comment:
1. `moment(timestamp)` can/should be shared 
2. maybe remove the `Csp` from the name (similar to `ColumnNameWithTooltip` - `TimestampTableCell`) 
3. maybe rename `timestamp` prop to `value` (`<TimestampTableCell value={x} />` reads nicer IMO)

Reply:
1. done
2. done
3. i'll leave it as `timestamp` since its a `TimestampTableCell` and the columns pass data as `timestamp`